aspnetcore/mvc/views/razor.md

```diff
@@ -568,6 +568,17 @@ The code renders the following HTML:
 
 ::: moniker-end
 
+### @attribute
+
+The `@attribute` directive sets the given attribute on the generated view.
+
+```cshtml
+@attribute [Serializable]
```

Comment:
I'd use a different attribute than `[Serializable]` because I believe there's already a `[Serializable]` attribute on components resulting in duplicate attribute errors. Do `[Authorize]` since that ties in with our new auth bits shipping in preview6.